frontend/coalesce/src/pages/OpportunitiesSearchPage.vue

```diff
@@ -10,7 +10,7 @@
           </q-card-section>
           <!-- search bar -->
           <q-toolbar>
-            <q-input rounded outlined class="search-bar" v-model="filter" type="text" >
+           <q-input rounded outlined class="search-bar" v-on:input="filterOpportunities" v-model="filter" type="text" >
```

Comment:
debounce="500" can be added to hold off on filterOpportunities() calls for 500 milliseconds to avoid flooding the server with REST API calls while the user is typing.